tests/cpp/test_mma.cpp

```diff
@@ -697,91 +494,6 @@ TEST_P(HopperRS, SingleTileTransposed) {
   EXPECT_TRUE(at::allclose(cg_outputs[0], tref, 1e-5, 1e-5));
 }
 
-TEST_P(HopperRS, SingleTileWithTMALoad) {
-  Fusion fusion;
```

Comment:
We already have test that uses TMA for both load and store. There is no need for a test that only has load but not store.